src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,73 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[];
+  select?: number;
```

Comment:
The `select` prop is marked as optional, but it is used in the component without a default value. Consider providing a default value or handling the case where `select` is undefined to avoid potential issues.